### PR TITLE
Add list of plugins to hide from the plugin directory

### PR DESCRIPTION
**Pull request by KrauseFx**
Exclude plugins that are already part of fastlane

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @KrauseFx</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: fastlane/actions/plugin_scores.rb [hunk 1]
```diff
@@ -26,8 +26,10 @@ def self.fetch_plugins
           break if results.count == 0
 
           plugins += results.collect do |current|
+            next if self.hidden_plugins.include?(current['name'])
+
             Fastlane::Helper::PluginScoresHelper::FastlanePluginScore.new(current)
-          end
+          end.compact
```
Comment (taquitos) [inline fastlane/actions/plugin_scores.rb]: til: .compact

**Review by taquitos — APPROVED** [review]
Nice job @KrauseFx, 🐙 it

File: fastlane/actions/plugin_scores.rb (hunk 1 above)
Comment (KrauseFx) [inline fastlane/actions/plugin_scores.rb]: It's the best method, I'd recommend it

**Pull request merged**